### PR TITLE
Clean up defender demo

### PR DESCRIPTION
**Pull request by archigup**
Description
-----------
Minor changes to defender demo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c [hunk 1]
```diff
@@ -62,23 +62,28 @@ eMetricsCollectorStatus eGetNetworkStats( NetworkStats_t * pxOutNetworkStats )
...
+    if( xMetricsStatus != 0 )
+    {
+        LogError( ( "Failed to acquire metrics from FreeRTOS+TCP tcp_netstat utility." ) );
+        eStatus = eMetricsCollectorCollectionFailed;
     }
 
     /* Fill our response with values gotten from FreeRTOS+TCP. */
     if( eStatus == eMetricsCollectorSuccess )
     {
+        LogDebug( ( "Network stats read. Bytes received: %d, packets received: %d "
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c:80]: Missing comma
```suggestion
        LogDebug( ( "Network stats read. Bytes received: %d, packets received: %d, "
```

File: FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c [hunk 2]
```diff
@@ -62,23 +62,28 @@ eMetricsCollectorStatus eGetNetworkStats( NetworkStats_t * pxOutNetworkStats )
...
+                    "bytes sent: %d, packets sent: %d.",
+                    ( int ) xMetrics.xInput.uxByteCount,
+                    ( int ) xMetrics.xInput.uxPacketCount,
+                    ( int ) xMetrics.XOutput.uxByteCount,
+                    ( int ) xMetrics.XOutput.uxPacketCount ) );
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c:85]: Looks like these are unsigned so why cast to an int?

File: FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c [hunk 3]
```diff
@@ -62,23 +62,28 @@ eMetricsCollectorStatus eGetNetworkStats( NetworkStats_t * pxOutNetworkStats )
...
+    /* Get metrics from FreeRTOS+TCP tcp_netstat utility. */
+    xMetricsStatus = vGetMetrics( &xMetrics );
 
-        if( xMetricsStatus != 0 )
-        {
-            eStatus = eMetricsCollectorCollectionFailed;
-        }
+    if( xMetricsStatus != 0 )
+    {
+        LogError( ( "Failed to acquire metrics from FreeRTOS+TCP tcp_netstat utility." ) );
```
Comment (jainamt) [inline FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c:73]: would it make sense to log ```xMetricsStatus``` in this case to debug what went wrong?

File: FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c [hunk 4]
```diff
@@ -97,19 +103,19 @@ eMetricsCollectorStatus eGetOpenTcpPorts( uint16_t * pusOutTcpPortsArray,
...
+    xMetricsStatus = vGetMetrics( &xMetrics );
 
-        if( xMetricsStatus != 0 )
-        {
-            eStatus = eMetricsCollectorCollectionFailed;
-        }
+    if( xMetricsStatus != 0 )
+    {
+        LogError( ( "Failed to acquire metrics from FreeRTOS+TCP tcp_netstat utility. Status: %d.",
+                    (int) xMetricsStatus) );
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/metrics_collector.c:117]: I don't think this file is uncrustified.

**Review by yourslab — APPROVED** [review]

**Review by aggarg — APPROVED** [review]

**Pull request merged**